Objective-C/CBLDatabase.h

```diff
@@ -326,6 +326,11 @@ typedef NS_ENUM(uint32_t, CBLMaintenanceType) {
  configuration. Without given the database configuration, the default configuration that
  is equivalent to setting all properties in the configuration to nil will be used.
  
+ @Note This method will copy the database without changing the encryption key of the original
+ database. The encryption key specified in the given config is the encryption key used for both
+ the original and copied database. To change or add the encryption key for the copied database,
+ call Database.changeEncryptionKey(key) for the copy.
```

Comment:
Can you refer to the objective-C version of the method name?

Reply:
done